services/types/couchapp/lib/schemas/news.js

```diff
@@ -2,23 +2,62 @@ exports.v1 = {
     "id": "news",
     "$schema": "http://json-schema.org/draft-04/schema#",
     "version": "1.0",
+    "description": "news document structure",
+    "definitions": {
+      "field_url": {
+          "type": "object",
+          "patternProperties": {
+              "^([a-z]{2}|und)$": {
+                  "type": "array",
+                  "items": {
+                      "type": "string",
+                      "pattern": "\\\/\\\/(\\w+:{0,1}\\w*@)?(\\S+)(:[0-9]+)?(\\\/|\\\/([\\w#!:.?+=&%@!\\-\\\/]))?"
```

Comment:
Strange, the pattern is different than above; there are more backslashes...
\\\/\\\/(\\w+:{0,1}\\w*@)?(\\S+)(:[0-9]+)?(\\\/|\\\/([\\w#!:.?+=&%@!\\-\\\/]))?
instead of
\\/\\/(\\w+:{0,1}\\w*@)?(\\S+)(:[0-9]+)?(\\/|\\/([\\w#!:.?+=&%@!\\-\\/]))?